quickstart-sda.md

```diff
@@ -0,0 +1,239 @@
+
+## Authentication for users (mock or AAI)
+
+To interact with SDA services, users need to provide jw token authentication. Ultimately, tokens can be fetched by [LS-AAI](https://lifescience-ri.eu/ls-login/) upon user login to an OIDC relaying party (RP) service that is [registered with LS-AAI](https://spreg-legacy.aai.elixir-czech.org/). An example of such an RP service is the [sda-auth](https://github.com/neicnordic/sda-auth) service, which however not included in the present stack.
+
+For the purposes of the starter-kit, jw tokens can be issued by the included `mock-oidc` service and used for authentication, instead. The `mock-oidc` is a simple python implementation that mimics basic OIDC functionality of LS-AAI. It doesnot require user authentication and serves a valid token through its `/token` endpoint:
+
+```cmd
+token=$(curl -s -k https://<mock-oidc_domain_name>/tokens | jq -r '.[0]')
+```
+
+This token is created upon deployment, see `scripts/make_credentials.sh` for more details. Note that the api actually returns a list of tokens where the first element is the token of interest and the rest are legacy tokens for [testing  `sda-download`](https://github.com/neicnordic/sda-download/blob/main/dev_utils/README.md#get-a-token).
+
+## How to perform common user tasks
+
+### Data encryption
+
+The `sda-pipeline` only ingests files that are encrypted with the archive's `c4gh` public key.For instance, using the go implementation of the [crypt4gh utility](https://github.com/neicnordic/crypt4gh) a file can be encrypted simply by running:
+
+```bash
+crypt4gh encrypt -f <file-to-encrypt> -p <sda-c4gh-public-key>
+```
+
+
+### Uploading data
+
+Users can upload data to the SDA by transferring them directly to the archive's `s3inbox` with an S3 client tool such as [s3cmd](https://s3tools.org/s3cmd):
+
+```bash
+s3cmd -c s3cmd.conf put <path-to-file.c4gh> s3://<username>/<target-path-to-file.c4gh>
+```
+
+where `s3cmd.conf` is a configuration file with the following content:
+```json
+[default]
+access_key = <USER_NAME>
+secret_key = <USER_NAME>
+access_token=<JW_TOKEN>
+check_ssl_certificate = False
+check_ssl_hostname = False
+encoding = UTF-8
+encrypt = False
+guess_mime_type = True
+host_base = <S3_INBOX_DOMAIN_NAME>
+host_bucket = <S3_INBOX_DOMAIN_NAME>
+human_readable_sizes = true
+multipart_chunk_size_mb = 50
+use_https = True
+socket_timeout = 30
+```
+
+### The sda-cli tool
+
+Instead of the tools above, users are **encouraged** to use the [sda-cli](https://github.com/NBISweden/sda-cli) which is a tool specificaly developed to perform all common SDA user related tasks in a convenient and unified manner. It is recommended to use precompiled executables for sda-cli which can be found at https://github.com/NBISweden/sda-cli/releases .
```

Comment:
```suggestion
Instead of the tools above, users are **encouraged** to use [`sda-cli`](https://github.com/NBISweden/sda-cli), which is a tool specifically developed to perform all common SDA user-related tasks in a convenient and unified manner. It is recommended to use precompiled executables for `sda-cli` which can be found at https://github.com/NBISweden/sda-cli/releases
```